packages/@sfx/sayt-plugin/src/sayt-plugin.ts

```diff
@@ -0,0 +1,21 @@
+import { Plugin, PluginRegistry, PluginMetadata } from '@sfx/core';
+import { Sayt, SaytConfig } from 'sayt';
+
+export default class SaytPlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'sayt',
+      depends: [],
+    };
+  }
+
+  sayt: Sayt;
+
+  constructor(options?: SaytConfig) {
+    this.sayt = new Sayt(options);
+  }
+
+  register() {
```

Comment:
If possible you should create a return type that describes the return value (the exposed value) type.